packages/deployer/src/config-data/config.json.example

```diff
@@ -1,42 +1,26 @@
 {
   "id": "e08f5610-5772-4986-92fb-c8aef5aee6a0",
   "nodeSettings": {
-    "providerIdShort": "9e5a89d",
-    "nodeVersion": "0.1.0",
     "cloudProvider": "aws",
-    "stage": "dev",
-    "region": "us-east-1",
     "logFormat": "plain",
+    "nodeVersion": "1.0.0",
```

Comment:
It would nice to eventually have a single config.json example, otherwise they get out of date/sync. Not really sure how we could do that though

Reply:
I still think that for the deployer we should pre-build with an arbitrary `config.json` and replace that at deployment-time with the user's validated `config.json`. I can use any `config.json` for that, it wouldn't matter. I'm waiting for the validator to mature and stop using the Serverless plugin for packing so there is still time.